pkg/virt-controller/watch/pool.go

```diff
@@ -1215,9 +1219,15 @@ func (c *PoolController) updateStatus(origPool *poolv1.VirtualMachinePool, vms [
 	}
 
 	pool.Status.Replicas = int32(len(vms))
+	pool.Status.ReadyReplicas = 0
+	for _, vm := range vms {
+		if vm.Status.Ready {
+			pool.Status.ReadyReplicas++
```

Comment:
You should be able to use the cache.  Search the codebase for ` filterReadyVMIs` to see an example.